store/LocalStore.js

```diff
@@ -24,7 +25,13 @@ LocalStore.prototype.incr = function (ip, cb, timeWindow, max) {
     ++current.current
 
     // Reset TLL if max has been exceeded and `continueExceeding` is enabled
```

Comment:
This comment is about continue exceeding, please keep the original order here:

it should be:
```
  if (this.continueExceeding)
  else if (this.exponentialBackoff)
  else
```

Reply:
Done